components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -89,6 +103,63 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem {
   }
 
   _selectedItem = selectedItem;
+  if (itemIndex == NSNotFound) {
+    return;
+  }
+
+  UIView *itemView = self.itemViews[itemIndex];
+  if ([itemView isKindOfClass:[MDCTabBarViewItemView class]]) {
+    MDCTabBarViewItemView *selectedItemView = (MDCTabBarViewItemView *)itemView;
+    selectedItemView.iconImageView.image = selectedItem.image;
```

Comment:
Does selecting an item deselects other items? 
Is multiple selection allowed?

Reply:
I think it should be OK. Since we're not changing images _specifically_ for selection, it isn't necessary in this PR.  However, I agree we'll have to handle deselection later.